AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -208,7 +208,7 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
     }
 
     get displayEmailField(): boolean {
-        return !!this.judge && this.judge.is_courtroom_account;
+        return !!this.judge && !ParticipantModel.IsEmailEjud(this.judge.email);
```

Comment:
can you add the new method IsEmailEjud to the test

Reply:
added tests to test static method returning as expected.